### ISSUE TITLE
Legitimate Site Blocked] binances.club

### ISSUE THREAD

**Issue by raomohsin39** [MetaMask/eth-phishing-detect#7531]
Hello dear devenblake, Please unblock the site of our community. We will be very grateful

**Comment by devenblake** [comment]
Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/6696.
